app/src/js/utils/table-config/reconciliation-reports.js

```diff
@@ -14,9 +14,8 @@ export const tableColumns = ({ dispatch, isGranules, query }) => ([
       const link = (location) => ({ pathname: `/reconciliation-reports/report/${encodeURIComponent(value)}`, search: getPersistentQueryParams(location) });
       switch (type) {
         case 'Internal':
-          return <Link to={link} onClick={(e) => handleDownloadClick(e, value, dispatch)}>{value}</Link>;
         case 'Granule Inventory':
-          return <Link to={link} onClick={(e) => handleCsvDownloadClick(e, value, dispatch)}>{value}</Link>;
+          return <Link to={link} onClick={(e) => handleDownloadClick(e, value, dispatch)}>{value}</Link>;
```

Comment:
Since you replaced `handleCsvDownloadClick` with the `handleDownloadClick`, can that be replaced everywhere and the function removed also?

Reply:
handleCsvDownloadClick has been removed.  handleDownloadCsvClick is a different function

Reply:
Oh, that's tricky naming, I didn't even see that. Ok, thanks for looking. I'll approve this.